dino.icu.yaml

```diff
@@ -331,6 +331,12 @@ tejas: # by tej
   type: CNAME
   value: tejasag.netlify.app.
 
+vansh:
+ - ttl: 1
+   type: CNAME
+   value: lowkeyvansh.github.io/web.
```

Comment:
You need to remove the "/web" from this line

Reply:
updated^